mlflow/langchain/api_request_parallel_processor.py

```diff
@@ -77,114 +76,6 @@ def increment_num_api_errors(self):
-                f"Unrecognized chat message role: {self.role}"
-            )
-
-    @staticmethod
-    def get_schema():
-        return Schema([ColSpec(DataType.string, "role"), ColSpec(DataType.string, "content")])
-
-
-class _ChatDeltaMessage(pydantic.BaseModel):
-    role: str
-    content: str
-
-
-# NB: Even though _ChatRequest is only referenced in one method within this module
-# (as of 12/27/2023), it must be defined at the module level for compatibility with
-# pydantic < 2
-class _ChatRequest(pydantic.BaseModel, extra="forbid"):
-    messages: List[_ChatMessage]
-
-
-class _ChatChoice(pydantic.BaseModel, extra="forbid"):
-    index: int
-    message: _ChatMessage = None
-    finish_reason: Optional[str] = None
-
-    @staticmethod
-    def get_schema():
-        return Schema(
-            [
-                ColSpec(DataType.integer, "index"),
-                ColSpec(_ChatMessage.get_schema(), "message", required=False),
-                ColSpec(DataType.string, "finish_reason", required=False),
-            ]
-        )
-
-
-class _ChatChoiceDelta(pydantic.BaseModel):
-    index: int
-    finish_reason: Optional[str] = None
-    delta: _ChatDeltaMessage
-
-
-class _ChatUsage(pydantic.BaseModel, extra="forbid"):
-    prompt_tokens: Optional[int] = None
-    completion_tokens: Optional[int] = None
-    total_tokens: Optional[int] = None
-
-    @staticmethod
-    def get_schema():
-        return Schema(
-            [
-                ColSpec(DataType.integer, "prompt_tokens", required=False),
-                ColSpec(DataType.integer, "completion_tokens", required=False),
-                ColSpec(DataType.integer, "total_tokens", required=False),
-            ]
-        )
-
-
-class _ChatResponse(pydantic.BaseModel, extra="forbid"):
-    id: Optional[str] = None
-    object: Literal["chat.completion"] = "chat.completion"
-    created: int
-    # Make the model field optional since we may not be able to get a stable model identifier
-    # for an arbitrary LangChain model
-    model: Optional[str] = None
-    choices: List[_ChatChoice]
-    usage: _ChatUsage
-
-    @staticmethod
-    def get_schema():
-        return Schema(
-            [
-                ColSpec(DataType.string, "id", required=False),
-                ColSpec(DataType.string, "object"),
-                ColSpec(DataType.integer, "created"),
-                ColSpec(DataType.string, "model", required=False),
-                ColSpec(Array(_ChatChoice.get_schema()), "choices"),
-                ColSpec(_ChatUsage.get_schema(), "usage"),
-            ]
-        )
-
-
-class _ChatChunkResponse(pydantic.BaseModel):
-    id: Optional[str] = None
-    object: Literal["chat.completion.chunk"] = "chat.completion.chunk"
-    created: int
-    # Make the model field optional since we may not be able to get a stable model identifier
-    # for an arbitrary LangChain model
-    model: Optional[str] = None
-    choices: List[_ChatChoiceDelta]
-
-
 @dataclass
 class APIRequest:
```

Comment:
Can we add docstring for the parameters? Some args meaning is not clear e.g. `convert_chat_responses` vs `dis_perform_chat_conversion`.....